.github/packager/packager.sh

```
@@ -4,19 +4,30 @@
     sed -i "$1" "$2"
   fi
 }
 
 ROOT_DIR="$(pwd)"
+
+# -----------------------------------------------------------
+# CRITICAL FIX: Since the script is executed from
+# /mnt/c/Portfolio/aerium/.github/packager, the 'pwd' command
+# returns the wrong root. We need to go up two directories.
+# -----------------------------------------------------------
+ROOT_DIR=$(dirname $(dirname "$ROOT_DIR"))
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)